src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Arrays;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    private static int totalTask = 0;
+    private static Task[] taskList = new Task[100];
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        totalTask++;        
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Looks Good, but you could declare this as a constant?

Reply:
Thanks for pointing it out